jmx/src/main/java/org/jboss/as/jmx/RemotingConnectorAdd.java

```diff
@@ -47,6 +48,11 @@ private RemotingConnectorAdd() {
     }
 
     @Override
+    protected boolean requiresRuntime(OperationContext context) {
```

Comment:
@luck3y Once this is in can you launch an embedded HC and also an admin-only embedded server and check what sockets they open? Should be zero, but it's worth checking whether we let other things slip through like this one did.